packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -235,6 +235,10 @@ export function useBlocksStore() {
       matchNode: (node) => node.type === 'paragraph',
       isInBlocksSelector: true,
       handleEnterKey(editor) {
+        /**
+         * We need to keep track of the initial position of the cursor
+         */
```

Comment:
these `/** */` should be used more for the a block or multiple(bigger) lines of comments in general. Could you please update it with single line comment `//` at all other places in this function.

IMO Even having 2-3 lines with `//` comment is fine to have inside a block.